app/assets/stylesheets/application.css.scss

```diff
@@ -33,7 +33,7 @@ body {
   padding-right: 30px;
   color: white;
   background-color: #263645;
-	background-image: url('http://timteeling.com/dev/img/network.svg'), radial-gradient(200px 500px at center, #636E79 0%, rgba(0,0,0,0) 50%, rgba(0,0,0,0));
+	background-image: url('http://timteeling.com/dev/img/network.svg'), radial-gradient(200px 1000px at center, #636E79 0%, rgba(0,0,0,0) 50%, rgba(0,0,0,0));
```

Comment:
Color literals like `#636E79` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#636E79` should be written as `#636e79`<br>Line should be indented with spaces, not tabs<br>Commas in function arguments should be followed by a single space<br>Prefer double-quoted strings<br>URL `http://timteeling.com/dev/img/network.svg` should not contain protocol or domain